File: ios/Porchivo/Config/AppConfig.swift
//
//  AppConfig.swift
//  Porchivo
//
//  Founder control panel — mirrors expo/config/app.ts and android AppConfig.kt.
//  Prices are DISPLAY labels only; real prices live in App Store Connect.
//

import Foundation

enum AppConfig {
    enum Pricing {
        static let monthlyDisplay = "$13.99"
        static let monthlyPerMonth = "$13.99/mo"
        static let monthlyProductID = "premium_monthly"

        static let annualDisplay = "$99.99"
        static let annualPerMonth = "$8.33/mo"
        static let annualProductID = "premium_annual"
        static let annualTrialDays = 7
        static let annualSavingsLabel = "Save 40%"

        static let lifetimeDisplay = "$500"
        static let lifetimeProductID = "porchivo_lifetime"

        static let winbackLabel = "Save 40% for 3 months"
        static let winbackDisplay = "$7.99/mo"
    }

    enum FamilyPlan {
        static let maxMembers = 5
        static let monthlyDisplay = "$23.99"
        static let monthlyProductID = "family_monthly"
        static let annualDisplay = "$179.99"
        static let annualPerMonth = "$15.00/mo"
        static let annualSavingsLabel = "Save 37%"
        static let annualProductID = "family_annual"
        static let annualTrialDays = 7
    }

    enum FreeLimits {
        /// Free users can track 1 package — the second hits the upgrade wall.
        static let maxPackages = 1
    }

    enum RiskThresholds {
        static let high = 65
        static let medium = 35
        static let baseScore = 30
    }

    enum Support {
        static let email = "support@porchivo.com"
        static let websiteURL = "https://porchivo.com"
        static let privacyPolicyURL = "https://porchivo.com/privacy"
        static let termsURL = "https://porchivo.com/terms"
    }

    enum SocialProof {
        static let packagesStolenStat = "119M"
        static let stolenRatio = "1 in 5"
    }
}

File: android/app/src/main/java/com/rork/porchivo/config/AppConfig.kt
package com.rork.porchivo.config

/**
 * Mirrors expo/config/app.ts — the founder control panel.
 * Prices are DISPLAY labels only; real prices live in Google Play Console.
 */
object AppConfig {

    object Pricing {
        const val MONTHLY_DISPLAY = "$13.99"
        const val MONTHLY_PER_MONTH = "$13.99/mo"
        const val MONTHLY_PRODUCT_ID = "premium_monthly"

        const val ANNUAL_DISPLAY = "$99.99"
        const val ANNUAL_PER_MONTH = "$8.33/mo"
        const val ANNUAL_PRODUCT_ID = "premium_annual"
        const val ANNUAL_TRIAL_DAYS = 7
        const val ANNUAL_SAVINGS_LABEL = "Save 40%"

        const val LIFETIME_DISPLAY = "$500"
        const val LIFETIME_PRODUCT_ID = "porchivo_lifetime"

        const val WINBACK_LABEL = "Save 40% for 3 months"
        const val WINBACK_DISPLAY = "$7.99/mo"
    }

    object FamilyPlan {
        const val MAX_MEMBERS = 5
        const val MONTHLY_DISPLAY = "$23.99"
        const val MONTHLY_PRODUCT_ID = "family_monthly"
        const val ANNUAL_DISPLAY = "$179.99"
        const val ANNUAL_PER_MONTH = "$15.00/mo"
        const val ANNUAL_SAVINGS_LABEL = "Save 37%"
        const val ANNUAL_PRODUCT_ID = "family_annual"
        const val ANNUAL_TRIAL_DAYS = 7
    }

    object FreeLimits {
        /** Free users can track 1 package — the second hits the upgrade wall. */
        const val MAX_PACKAGES = 1
    }

    object RiskThresholds {
        const val HIGH = 65
        const val MEDIUM = 35
        const val BASE_SCORE = 30
    }

    object Support {
        const val EMAIL = "support@porchivo.com"
        const val WEBSITE_URL = "https://porchivo.com"
        const val PRIVACY_POLICY_URL = "https://porchivo.com/privacy"
        const val TERMS_URL = "https://porchivo.com/terms"
    }

    object SocialProof {
        const val PACKAGES_STOLEN_STAT = "119M"
        const val STOLEN_RATIO = "1 in 5"
    }
}
